src/traces/bar/attributes.js

```diff
@@ -42,7 +42,15 @@ var marker = extendFlat({
         editType: 'style',
         description: 'Sets the opacity of the bars.'
     },
-    pattern: pattern
+    pattern: pattern,
+    cornerradius: {
+        valType: 'any',
+        editType: 'calc',
+        description: [
+            'Sets the rounding of corners. May be an integer number of pixels,',
+            'or a percentage of bar width(as a string). Defaults to `layout.barcornerradius`.'
```

Comment:
```suggestion
            'or a percentage of bar width (as a string ending in %). Defaults to `layout.barcornerradius`.',
            'In stack or relative barmode, the first trace to set cornerradius is used for the whole stack.'
```

This tripped me up a little when testing - makes total sense to do it this way and you got the behavior exactly right AFAICT, but I think it's worth describing here.

Reply:
You're right, this should be documented. Thanks @alexcjohnson 🙏